Car/CarRental.Tests/CarRentalTests.cs

```diff
@@ -0,0 +1,134 @@
+using CarRental.Domain;
+using CarRental.Domain.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace CarRental.Tests;
+
+/// <summary>
+/// Unit tests for car rental queries using DataSeeder fixture.
+/// </summary>
+public class RentalTests(DataSeeder data) : IClassFixture<DataSeeder>
+{
+    /// <summary>
+    /// Customers who have rented a Toyota Corolla, sorted by full name.
+    /// </summary>
+    [Fact]
+    public void ClientsByModelShouldReturnClientsOrderedByFullName()
+    {
+        var clients = data.Rentals
+            .Where(r => r.Car?.Generation?.Model?.Name == "Toyota Corolla")
```

Comment:
После объявления required все вопросы можно откинуть.
```suggestion
            .Where(r => r.Car.Generation.Model.Name == "Toyota Corolla")
```